documentation/docs/how_tos/outputs_and_transactions/06_destroy_alias_output.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Destroy an Alias Output
+description: 'How to destroy an alias output using wallet.rs'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- outputs
+- destroy
+- destroy alias output
+- nodejs
+- python
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../_admonitions/_guide_coming_soon.md";
+import NodejsOutput from "../../_admonitions/_guide_coming_soon.md";
+import PythonCode from "../../_admonitions/_guide_coming_soon.md";
+import PythonOutput from "../../_admonitions/_guide_coming_soon.md";
+import RustCode from  "../../libraries/rust/how_to/outputs_and_transactions/_16_destroy_alias_output/_code_sample.mdx";
+import RustOutput from  "../../libraries/rust/how_to/outputs_and_transactions/_16_destroy_alias_output/_expected_output.md";
+
+You can destroy an alias output by ID using the `Account.destroy_alias(alias_id, options)` function. Any outputs
+controlled by the alias will be swept unless they have a
+[storage deposit return](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/simple_transfers#storage-deposit-returns-in-transfers),
+[time-lock](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/simple_transfers#timelocking-funds-in-transfers)
+or [expiration unlock](https://wiki.iota-community.org/shimmer/introduction/develop/explanations/ledger/simple_transfers#transfer-of-funds-with-expiration)
+conditions.
```

Comment:
I'm currently about to change this https://github.com/iotaledger/wallet.rs/pull/1440
If there are outputs owned by the alias, then the user will get an error, because he should take care of them himself before